reactApp/src/data/putRating.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const putRating = async (term, remedy, formValues) => {
+  try {
+    const response = await axios.put(
+      `http://localhost:7000//remedies/${term}`,
+      {
+        ...formValues,
+        userId: "[USER.ID]",
+        remedyId: remedy.id,
+        remedyName: remedy.remedyName,
+      }
```

Comment:
At the moment, it seems you're only sending this in the body: 
{
	"userId": "[USER.ID]"
}

Could you add the rating value there? I think it was there before your changes:) 
Also, you can delete remedy name, since we have the id